src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationService.kt

```diff
@@ -96,6 +97,15 @@ class ActivitiesMigrationService(
         }
   }
 
+  suspend fun endMigratedActivities(migrationId: String) {
+    val activityCount = activitiesMappingService.getMigrationCount(migrationId)
+    if (activityCount == 0L) throw NotFoundException("No migrations found for $migrationId")
+
+    val allActivityIds = activitiesMappingService.getActivityMigrationDetails(migrationId, activityCount).content
+      .map { it.nomisCourseActivityId }
+    nomisApiService.endActivities(allActivityIds)
```

Comment:
This is the main change - grab all NOMIS course acitivity ids that were migrated and call the NOMIS API to end those activities.